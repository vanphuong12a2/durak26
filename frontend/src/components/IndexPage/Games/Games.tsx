import React from 'react';

export interface GamesProps {
  playing: boolean
}

const Games = (props: GamesProps) => {
  const dummyHandler = () => {;};
  return (
    <div className='new-game'>
      <a className='waves-effect waves-light btn-small' onClick={dummyHandler}>New game</a>
    </div>
  );
};

export default Games;
