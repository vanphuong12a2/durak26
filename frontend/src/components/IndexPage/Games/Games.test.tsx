import React from 'react';
import {create} from 'react-test-renderer';
import Games from './Games';

describe('<Games />', () => {

  describe('Snapshots', () => {
    it('renders correctly', () => {
      const component = create(<Games playing={false}/>);
      expect(component.toJSON()).toMatchSnapshot();
    });

    it('renders playing game correctly', () => {
      const component = create(<Games playing={true}/>);
      expect(component.toJSON()).toMatchSnapshot();
    });
  });

});
