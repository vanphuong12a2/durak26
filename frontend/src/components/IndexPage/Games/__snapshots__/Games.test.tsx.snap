// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Games /> Snapshots renders correctly 1`] = `
<div
  className="new-game"
>
  <a
    className="waves-effect waves-light btn-small"
    onClick={[Function]}
  >
    New game
  </a>
</div>
`;

exports[`<Games /> Snapshots renders playing game correctly 1`] = `
<div
  className="new-game"
>
  <a
    className="waves-effect waves-light btn-small"
    onClick={[Function]}
  >
    New game
  </a>
</div>
`;
